bot/cogs/pavlov.py

```diff
@@ -9,10 +9,11 @@
 
 import aiohttp
 import discord
-from discord.ext import commands
+from discord.ext import tasks, commands
```

Comment:
Is the `tasks` extension used anywhere here? Generally I would advise against it, and instead use asyncio tasks directly.